resources/js/routes.js

```diff
@@ -46,4 +47,10 @@ export default [
     component: Admin,
     meta: { title: 'iSTART | Admin', requiresAuth: true, adminAuth: true }
   },
+  {
+    path: '/intro',
```

Comment:
this doesn't need to exist because this will be loaded when sliding through the object

Reply:
o yes thank u